media/css/cms/flare-navigation.css

```diff
@@ -0,0 +1,488 @@
+    background: var(--fl-theme-navigation-bg);
+}
+
+.fl-header a {
+    text-decoration: none;
+}
+
+.fl-header a:hover {
+    text-decoration: underline;
+}
+
+.fl-header ul {
+    list-style: none;
+}
+
+.fl-header-inner {
+    max-width: calc(var(--token-widths-viewport-wrapper-screen-2xl) * 1px);
+    margin: 0 auto;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
+/* ================================================
+   Logo
+   ================================================ */
+
+.fl-logo-fx {
+    width: 120px;
+    height: 40px;
+    margin: 0;
+    text-indent: -9999px;
+    overflow: hidden;
+    white-space: nowrap;
+    background: url('/media/img/logos/firefox/logo-word-hor.svg') no-repeat;
+    background-size: 120px 40px;
+    text-decoration: none;
+    z-index: 1;
+}
+
+/* ================================================
+   Hamburger Menu
+   ================================================ */
+
+.fl-hamburger {
+    width: 32px;
+    height: 32px;
+    display: inline-flex;
+    align-items: center;
+    justify-content: center;
+    position: relative;
+    background: transparent;
+    background-image: linear-gradient(var(--fl-theme-navigation-icon), var(--fl-theme-navigation-icon));
+    background-repeat: no-repeat;
+    background-position: left 12px center;
+    background-size: 16px 2px;
+    border: none;
+    border-radius: calc(var(--token-scale-4) * 1px);
+    cursor: pointer;
+    transition: background-size var(--token-transition-base);
+}
+
+.fl-hamburger::before,
+.fl-hamburger::after {
+    content: "";
+    position: absolute;
+    left: 12px;
+    right: 4px;
+    height: 2px;
+    background: var(--fl-theme-navigation-icon);
+    border-radius: 1px;
+    transition: transform var(--token-transition-base), top var(--token-transition-base), bottom var(--token-transition-base);
+}
+
+.fl-hamburger::before {
+    top: 9px;
+}
+
+.fl-hamburger::after {
+    bottom: 9px;
+}
+
+.fl-hamburger[aria-expanded="true"] {
+    background-size: 0 0;
+}
+
+.fl-hamburger[aria-expanded="true"]::before {
+    top: 50%;
+    transform: translateY(-50%) rotate(45deg);
+}
+
+.fl-hamburger[aria-expanded="true"]::after {
+    bottom: auto;
+    top: 50%;
+    transform: translateY(-50%) rotate(-45deg);
+}
+
+/* ================================================
+   Show Mobile Menu Button
+   ================================================ */
+
+.fl-show-mobile-menu {
+    font-weight: var(--token-font-weight-semibold);
+    text-decoration: underline;
+    height: 32px;
+    margin: 0;
+    padding: 0;
+    background: transparent;
+    color: var(--fl-theme-color-text-body);
+}
+
+/* ================================================
+   Navigation
+   ================================================ */
+
+.fl-nav {
+    display: none;
+    flex-grow: 1;
+    background: var(--fl-theme-navigation-bg);
+}
+
+.fl-nav a[href^='http']::after {
```

Comment:
We've already got icon components that can be used here, instead of duplicating the CSS.